SocialFilter/sections/specific.txt

```diff
@@ -590,6 +590,7 @@ fbs.co.jp##.newsDetailAssets__sns
 locipo.jp##div[class^="share_"]
 news.mingpao.com##.articlesharepart
 thurrott.com#%#//scriptlet('remove-in-shadow-dom','button[data-testid="ShareBtn"]')
```

Comment:
Please add the domain to the existing rule
`metro.co.uk,thurrott.com#%#//scriptlet('remove-in-shadow-dom', 'button[data-testid="ShareBtn"]')`